meteoblue_dataset_sdk/caching/filecache.py

```diff
@@ -0,0 +1,105 @@
+import datetime
+import hashlib
+import json
+import logging
+import os
+import tempfile
+import zlib
+from typing import Union, Optional
+
+import aiofiles
+from aiofiles import os as aios
+
+from .cache import Cache
+
+CACHE_DIR = "mb_cache"
+# 7200s == 2h
+DEFAULT_CACHE_DURATION = 7200
+
+
+class FileCache(Cache):
+    def __init__(
+        self,
+        cache_path: Optional[str] = None,
+        cache_ttl: int = DEFAULT_CACHE_DURATION,
+        compression_level: int = 6,
+    ):
+        if cache_path is None:
+            cache_path = tempfile.gettempdir()
+        cache_path = os.path.join(cache_path, CACHE_DIR)
```

Comment:
Why is  `mb_cache` appended? If the user  supplied a  cache directory like `/mnt/nvme/meteoblue/`, it should write cache objects in  like `/mnt/nvme/meteoblue/h231/34h2e2b23b2`

Using mb_cache only as a prefix, if the  default  gettempdir function is used, is ok.

Reply:
yeah i can